protocol/x/perpetuals/keeper/perpetual_test.go

```diff
@@ -142,6 +143,7 @@ func TestCreatePerpetual_Failure(t *testing.T) {
 		atomicResolution  int32
 		defaultFundingPpm int32
 		liquidityTier     uint32
+		marketType        types.PerpetualMarketType
```

Comment:
Add a test for an invalid market type leading to the `CreatePerpetual` call failing?

Reply:
done